Template/Tag/MatomoTag.web.js

```diff
@@ -14,6 +14,13 @@
     };
 
     window._paq = window._paq || [];
+    // Store the initial state of window._paq so that we can apply it to all of the configs
+    var initialPaq = window._paq && window._paq.length ? JSON.parse(JSON.stringify(window._paq)) : initialPaq || [];
```

Comment:
@snake14 Can you add a comment why do we need to stringify and convert it back to JSON here ?

Reply:
Sure. I'm adding a comment now.